features/kinderen.feature

```diff
@@ -31,44 +31,53 @@ Functionaliteit: Kinderen van een persoon raadplegen
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999996150                       |
+      | burgerservicenummer | 000001234                       |
       | fields              | kinderen.naam                   |
-      Dan heeft de response een persoon met een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde  |
-      | voornamen     | William |
-      | geslachtsnaam | Postma  |
-      En heeft de persoon een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde     |
-      | voornamen     | Sebastiaan |
-      | voorvoegsel   | de         |
-      | geslachtsnaam | Boer       |
-      En heeft de persoon een 'kind' met de volgende 'naam' gegevens
-      | naam          | waarde |
-      | voornamen     | Walter |
-      | voorvoegsel   | de     |
-      | geslachtsnaam | Boer   |
+      Dan heeft de response een persoon met een 'kind' met de volgende gegevens
+      | naam               | waarde  |
+      | type               | Kind    |
```

Comment:
kind is niet polymorfisch, daardoor heeft hij geen type. De oude Dan stappen zijn correct. Alleen de 'voorletters' veld moet daar worden toegevoegd.

Reply:
De proxy levert het veld 'type' met de waarde 'Kind' wel. Dus daar moet dan ook een aanpassing plaatsvinden.

Reply:
> De proxy levert het veld 'type' met de waarde 'Kind' wel. Dus daar moet dan ook een aanpassing plaatsvinden.

Weet je dat zeker? Het staat namelijk niet in de OAS specificatie

Reply:
Zie hier:

![image](https://user-images.githubusercontent.com/20185784/188911057-0f0dd14b-15f0-479a-83db-481c34e9ba19.png)

Reply:
Zo te zien is jouw container image niet up-to-date. Kan je de volgende statement uitvoeren en kijken of je dan andere resultaten krijgt?

```
docker-compose -f docker-compose-e2e.yml build
```

Reply:
Nu komt

`"type":  "Kind"`

niet meer voor.